cmd/lncli/commands.go

```diff
@@ -111,13 +111,45 @@ var SendCoinsCommand = cli.Command{
 }
 
 func sendCoins(ctx *cli.Context) error {
+	var addr string = ""
```

Comment:
Minor nit: you don't need to declare the types of any of the variables if you use the `:=` operator. 

So this fragment can be reduce to (in the style of the rest of the codebase: 
```go
var (
    amt int64
    addr string
)

args := tx.Args()
```